ra/ra.go

```diff
@@ -2435,6 +2435,16 @@ func (ra *RegistrationAuthorityImpl) DeactivateAuthorization(ctx context.Context
 	if _, err := ra.SA.DeactivateAuthorization2(ctx, &sapb.AuthorizationID2{Id: authzID}); err != nil {
 		return nil, err
 	}
+	if req.Status == string(core.StatusPending) {
+		// Deactivating a pending authz has no security benefits and prevents us
+		// from reusing it and any orders associated with it. Some clients appear to
+		// accidentally abuse this behavior, so treat deactivating a pending authz
+		// the same as failing validation.
+		err = ra.countFailedValidations(ctx, req.RegistrationID, identifier.NewDNS(req.DnsName))
+		if err != nil {
+			ra.log.Warningf("incrementing failed validations: %s", err)
```

Comment:
This log line, if we keep it, should be differentiated from the identical error in RegistrationAuthorityImpl.PerformValidation, this would make the context easier to identify.